src/icmp/IcmpSquid.cc

```diff
@@ -182,6 +182,9 @@ IcmpSquid::DomainPing(Ip::Address &to, const char *domain)
 #if USE_ICMP
     debugs(37, 4, HERE << "'" << domain << "' (" << to << ")");
     SendEcho(to, S_ICMP_DOM, domain, 0);
+#else
+    (void) to; /* arg not used */
+    (void) domain; /* arg not used */
```

Comment:
I think this code speaks for itself. And for consistency with the official code sake, let's remove the space.

```suggestion
    (void)to;
    (void)domain;
```

Same for similar changes, of course.

Reply:
removing